chirp/drivers/hg_uv98.py

```diff
@@ -33,6 +32,18 @@
 
 LOG = logging.getLogger(__name__)
 
+# Gross hack to handle missing future module on un-updatable
+# platforms like MacOS. Just avoid registering these radio
+# classes for now.
+try:
+    from builtins import bytes
+    has_future = True
+except ImportError:
+    has_future = False
```

Comment:
You need to make the registration conditional on this so we don't register the driver if it will crash later. See the bottom of tk8180.py.

Apologies that you based this on tk8102.py which has a build system exception and also needs fixing, but thanks for following up.

Reply:
@kk7ds alright, i think the latest change should be sufficient

ultimately i want this driver available everywhere, so i'm going to try to get rid of this import, but for now, lets get unblocked on #10110 with a new build

Reply:
in the meantime i've posted #104 which removes the `builtins` import entirely and should also fix the startup bug.